cockpit/src/Select/Select.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import $ from 'jquery';
```

Comment:
I'm not big fan of jQuery. Is it really necessary to use jQuery? Is it widely used in other cockpit plugins? Are we going to use it at other parts of our plugin? `$` is used 4 times.

Reply:
> Is it really necessary to use jQuery?

No, the alternative is to re-implement the component in pure React, as in https://github.com/cockpit-project/cockpit/blob/c1dd666129fd173e0c54ae62f2ecbd8d98907bf5/pkg/lib/cockpit-components-listing.jsx, but that is logic we don't necessarily wish to own or maintain (I tried pulling in this code but ran into issues as I described in cockpit-project/cockpit#7775). jQuery was the simplest solution to implement (and is less for our team to maintain); however, once patternfly/patternfly-react#54 is resolved, we'll switch to consuming whatever patternfly-react implements.

Reply:
Wow. You did lot of work and research about this. 👍 

I thing that jQuery can be used as temporary workaround.